src/components/Send.jsx

```diff
@@ -480,33 +480,36 @@ export default function Send() {
 
   return (
     <>
-      {isLoading ? (
-        <div className="d-flex justify-content-center align-items-center">
-          <rb.Spinner as="span" animation="border" size="sm" role="status" aria-hidden="true" className="me-2" />
-          {t('send.loading')}
-        </div>
-      ) : (
-        <div className="send">
-          <PageTitle title={t('send.title')} subtitle={t('send.subtitle')} />
+      <div className="send">
+        <PageTitle title={t('send.title')} subtitle={t('send.subtitle')} />
 
-          <rb.Fade in={!isCoinjoinOptionEnabled} mountOnEnter={true} unmountOnExit={true}>
+        <rb.Fade in={!isCoinjoinOptionEnabled} mountOnEnter={true} unmountOnExit={true}>
+          {serviceInfo?.makerRunning || serviceInfo?.coinjoinInProgress ? (
             <div className="mb-4 p-3 border border-1 rounded">
               <small className="text-secondary">
                 {serviceInfo?.makerRunning && t('send.text_maker_running')}
                 {serviceInfo?.coinjoinInProgress && t('send.text_coinjoin_already_running')}
               </small>
             </div>
-          </rb.Fade>
-
-          {alert && (
-            <rb.Alert className="slashed-zeroes" variant={alert.variant}>
-              {alert.message}
-            </rb.Alert>
+          ) : (
+            <></>
```

Comment:
Feels that this could be removed by someone not knowing the impacts in the future.  Do you think the whole content should be wrapped in `<></>` then?

Reply:
Good point. Changed!